plugins/configurationcache/configurationcachetree.cpp

```diff
@@ -828,6 +828,9 @@ int CacheTree::SaveCache(std::string filename)
 
     FILE* pfile;
     pfile = fopen(_fulldirname.c_str(),"wb");
+    if( pfile == NULL) {
+        return 0;
```

Comment:
Could you help add a mention of what return value means in the header (`configurationcachetree.h`)? Thanks.

Reply:
done.